src/index.js

```diff
@@ -0,0 +1,125 @@
+let ArrayList = {
+    array: [],
+    size: 0,
+    init: function(argument) {
+        if (!argument) return 'Please input an array';
+        ArrayList.array = argument;
+        ArrayList.size = argument.length;
+
+        return ArrayList.array
+    },
+    addAtEnd: function(arrElm) {
+        if (!arrElm)  return 'Please input an array';
+            const arr = ArrayList.array;
+            arr[arr.length] = arrElm;
+            ArrayList.size = arr.length;
+
+            return arr;
+    },
+    clearObject: function() {
+        ArrayList.array = [];
+        ArrayList.size = 0;
+
+        return ArrayList.array
+    },
+    clearFromEnd: function() {
+        const arr = ArrayList.array;
+        arr.length = [arr.length - 1];
+        ArrayList.size = arr.length;
+        
+        return ArrayList.array;
+    },
```

Comment:
I said that more correct will be name deleteFromEnd and don't forget an empty line after variables declaration.